sdk/evaluation/azure-ai-evaluation/README.md

```diff
@@ -20,88 +26,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
 
```

Comment:
should we call out the need to have an Azure OpenAI/OpenAI model deployment if using GPT-based evaluators or an Azure AI Project if using content safety evaluators? Following this example here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-blob/README.md?plain=1#L24C1-L25C119